packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -75,7 +75,8 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
 
   /// The index into [items] of the current active item.
   ///
-  /// Must not be null.
+  /// Must not be null and must inclusively be between 0 and the number of tabs
```

Comment:
FYI, the 2nd part of this statement could be shortened to "and must be in [0, tab count)", but maybe that's worse...

Reply:
I think it'd take slightly more parsing